internal/server/middleware.go

```diff
@@ -290,7 +290,7 @@ func reqBearerToken(c *gin.Context, opts Options) (string, error) {
 
 	// this will get caught by key validation, but check to be safe
 	if strings.TrimSpace(bearer) == "" {
-		return "", fmt.Errorf("%w: skipped validating empty token", internal.ErrUnauthorized)
+		return "", AuthenticationError{Message: "Authorization header is missing a bearer token"}
```

Comment:
This message will be sent to the user. It seems safe to let them know they have an invalid `Authorization` header, because that's user error and doesn't really leak any information.